Fix JDBC trace doc for DB2 (#7401)
OpenLiberty/docs: modules/ROOT/pages/jdbc-tracing.adoc

[#DB2JCC]
=== DB2 JCC

> The DB2 JCC does not support the `java.util.logging` library. This driver has it's own logging implementations which outputs directly to a file using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that will configure a PrintWriter that will output JCC driver trace to trace.log. The following `server.xml` file example shows you the configuration to enable trace for the DB2 JCC driver and specify a custom property.

Reviewer: ```suggestion
The DB2 JCC does not support the `java.util.logging` library. This driver has its own logging implementations that output directly to a file by using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that configures a PrintWriter to output JCC driver trace to the ` trace.log` file. The following `server.xml` file example shows you the configuration to enable trace for the DB2 JCC driver and specify a custom property.
```

[#InformixDB2JCC]
=== Informix using DB2 JCC

> Informix using the DB2 JCC driver does not support the `java.util.logging` library. This driver has it's own logging implementations which outputs directly to a file using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that will configure a PrintWriter that will output JCC driver trace to trace.log. The following `server.xml` file example shows you the configuration to enable trace for Informix using the DB2 JCC driver and specify a custom property.

Reviewer: ```suggestion
Informix using the DB2 JCC driver does not support the `java.util.logging` library. This driver has its own logging implementations that output directly to a file by using a PrintWriter. Open Liberty provides the `com.ibm.ws.db2.logwriter` log writer name for trace specification that configures a PrintWriter to output JCC driver trace to the `trace.log` file. The following `server.xml` file example shows you the configuration to enable trace for Informix by using the DB2 JCC driver and specify a custom property.
```
